src/Repository.js

```diff
@@ -70,8 +77,18 @@ export default class Repository {
     return this.version === "independent";
   }
 
-  buildPackageGraph() {
+  buildPackageGraph({scope, ignore}) {
+
+    // TODO: Replace these with a nicer config sieve.
+    if (this.constructor.name === "BootstrapCommand") {
```

Comment:
Yeah, that would be a nicer setup.  That's what I was getting at with the `TODO` above.

What I'd like is to be able to specify config for _any_ command, like:

```json
{
  "asini": "x.x.x",
  "version": "1.2.0",
  "commands": {
    "bootstrap": {
        "ignore": "test-package"
    },
  }
}
```

Figured I'd leave that for a dedicated patch, though, and do all of the commands at once.